braid/_braid_status.c

```diff
@@ -382,6 +392,40 @@ ACCESSOR_FUNCTION_GET4(Access, TILD,            Real, Int, Int, Int)
 ACCESSOR_FUNCTION_GET1(Access, WrapperTest,     Int)
 ACCESSOR_FUNCTION_GET1(Access, CallingFunction, Int)
 
```

Comment:
Could you add access to tupper and tlower to the Step function as well?  I have a suspicion that this will come in handy later.

Reply:
Added in https://github.com/XBraid/xbraid/pull/27/commits/7c22105086d4a0c86e514ee82db68039c31534b7

Reply:
Actually its not as simple as in that commit. Every StepStatusInit will need to get the local upper and lower times.

Reply:
Looks like its working now, addb4ab5ec2fbb34481dd5f5190a9740ca0aee22